scp.c

```diff
@@ -488,9 +489,44 @@ main(int argc, char **argv)
 	msetlocale();
 
 	/* Copy argv, because we modify it */
-	newargv = xcalloc(MAXIMUM(argc + 1, 1), sizeof(*newargv));
+	newargv = xcalloc(MAXIMUM(argc + 1, 1), sizeof(*newargv));	
+#ifdef WINDOWS	
```

Comment:
Please add comments why the ifdef block is required.
It will help when we go upstream.

Reply:
added commments

Reply:
Also mention the directory separator conversion "\\" to "/"

Reply:
I did mentioned it in the code